### ISSUE TITLE
Uploading larger file causes little yellow box to appear and fails, works on other computers though

### ISSUE THREAD

**Issue by JSmith-BitFlipper** [owncloud/core#12516]
I have a linux and windows system. On the linux system running Ubuntu 14.04 and on the windows 7 system, when I upload a larger file (more than a few MB), the file fails to upload and a little yellow box appears. However, I have two systems running Ubuntu 13.10 and they work fine. 

### Steps to reproduce
Log in to owncloud on either my Ubuntu 14.04 or windows system
Try to upload a file that is greater than a few MB

### Expected behaviour
The file should get uploaded

### Actual behaviour
Upload bar reaches the end and finished. Instead of the uploaded file appearing with the other files, a little yellow box (the message box) appears, except with no information. The file was not uploaded.

### Server configuration
**Operating system**:
Ubuntu 14.04.1 LTS
Kernel: Linux 3.13.0-37-generic #64-Ubuntu SMP Mon Sep 22 21:28:38 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
Openssl: OpenSSL 1.0.1f 6 Jan 2014

**Web server:**
Server version: Apache/2.4.7 (Ubuntu)
Server built:   Jul 22 2014 14:36:38

**Database:**
mysql  Ver 14.14 Distrib 5.5.40, for debian-linux-gnu (x86_64) using readline 6.3

**PHP version:**
PHP: version 5.5.9-1ubuntu4.5.

**ownCloud version:** (see ownCloud admin page)
7.0.3

**Updated from an older ownCloud or fresh install:**
Updated from 7.0.2, problem still persisted then. I hoped the update would fix it, but it does not.

**List of activated apps:**
None, only the default.

**The content of config/config.php:**

```
<?php
$CONFIG = array (
  'instanceid' => 'oc72d9451e9c',
  'passwordsalt' => 'hidden',
  'trusted_domains' => 
  array (
    0 => 'owncloud.servername.com',
  ),
  'datadirectory' => '/var/www/servername/subs/owncloud/data',
  'dbtype' => 'mysql',
  'version' => '7.0.3.4',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'cloud',
  'dbpassword' => 'hidden',
  'installed' => true,
  'forcessl' => true,
  'mail_smtpmode' => 'smtp',
  'mail_from_address' => 'admin',
  'mail_domain' => 'severname.com',
  'mail_smtphost' => 'mail.servername.com',
  'mail_smtpport' => '587',
  'mail_smtpsecure' => 'tls',
  'mail_smtpauth' => true,
  'mail_smtpauthtype' => 'LOGIN',
  'mail_smtpname' => 'admin@servername.com',
  'mail_smtppassword' => 'hidden',
  'loglevel' => '2',
  'theme' => '',
  'maintenance' => false,
);

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** yes/no
Yes, https

### Client configuration
**Browser:**
Firefox 33.0
Chromium Version 39.0.2171.65 Ubuntu 14.04 (64-bit)

On windows: Firefox 33.1

**Operating system:**
Ubuntu 14.04.1 LTS
Kernel: Linux 3.13.0-36-generic #63-Ubuntu SMP Wed Sep 3 21:30:07 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
Openssl: OpenSSL 1.0.1f 6 Jan 2014

On windows: windows 7

#### Web server error log
```
error.log
[Fri Nov 28 18:36:35.578059 2014] [ssl:warn] [pid 4223] AH01909: RSA certificate configured for owncloud.servername.com:443 does NOT include an ID which matches the server name
[Fri Nov 28 18:36:35.578390 2014] [ssl:warn] [pid 4223] AH02292: Init: Name-based SSL virtual hosts only work for clients with TLS server name indication support (RFC 4366)
[Fri Nov 28 18:36:35.578481 2014] [mpm_prefork:notice] [pid 4223] AH00163: Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.5 OpenSSL/1.0.1f configured -- resuming normal operations
[Fri Nov 28 18:36:35.578490 2014] [core:notice] [pid 4223] AH00094: Command line: '/usr/sbin/apache2'
[Sat Nov 29 18:33:41.980865 2014] [negotiation:error] [pid 18027] [client 107.191.34.156:38532] AH00687: Negotiation: discovered file(s) matching request: /var/www/servername.com/subs/owncloud/settings/admin (None could be negotiated)., referer: https://owncloud.servername.com/index.php/apps/files/
[Sat Nov 29 18:33:42.488126 2014] [access_compat:error] [pid 18026] [client 192.168.1.1:50611] AH01797: client denied by server configuration: /var/www/servername.com/subs/owncloud/data/htaccesstest.txt
[Sat Nov 29 18:33:44.443761 2014] [negotiation:error] [pid 18006] [client 107.191.34.156:38537] AH00687: Negotiation: discovered file(s) matching request: /var/www/servername.com/subs/owncloud/settings/ajax/checksetup (None could be negotiated)., referer: https://owncloud.servername.com/index.php/settings/admin

```

#### ownCloud log (data/owncloud.log)
```
{"reqId":"5479efaa4a247","app":"webdav","message":"Sabre\\DAV\\Exception\\NotAuthenticated: No basic authentication headers were found","level":0,"time":"2014-11-29T16:09:14+00:00","method":"PROPFIND","url":"\/remote.php\/webdav"}
{"reqId":"547a57e851448","app":"PHP","message":"Array to string conversion at \/home\/owncloud\/owncloud_data\/lib\/private\/l10n\/string.php#57","level":3,"time":"2014-11-29T23:34:00+00:00","method":"GET","url":"\/index.php\/apps\/updater\/update.php"}
{"reqId":"547a580ad0f17","app":"PHP","message":"Declaration of OCA\\TemplateEditor\\Controller\\AdminSettingsController::render() should be compatible with OCP\\AppFramework\\Controller::render($templateName, array $params = Array, $renderAs = 'user', array $headers = Array) at \/home\/owncloud\/owncloud_data\/apps\/templateeditor\/controller\/adminsettingscontroller.php#31","level":3,"time":"2014-11-29T23:34:34+00:00","method":"GET","url":"\/index.php\/apps\/templateeditor\/settings\/mailtemplate?theme=default&template=core%2Flostpassword%2Ftemplates%2Femail.php"}
{"reqId":"547a580f7f3b4","app":"PHP","message":"Declaration of OCA\\TemplateEditor\\Controller\\AdminSettingsController::render() should be compatible with OCP\\AppFramework\\Controller::render($templateName, array $params = Array, $renderAs = 'user', array $headers = Array) at \/home\/owncloud\/owncloud_data\/apps\/templateeditor\/controller\/adminsettingscontroller.php#31","level":3,"time":"2014-11-29T23:34:39+00:00","method":"GET","url":"\/index.php\/apps\/templateeditor\/settings\/mailtemplate?theme=default&template=core%2Ftemplates%2Faltmail.php"}
{"reqId":"547a5813b2d89","app":"PHP","message":"Declaration of OCA\\TemplateEditor\\Controller\\AdminSettingsController::render() should be compatible with OCP\\AppFramework\\Controller::render($templateName, array $params = Array, $renderAs = 'user', array $headers = Array) at \/home\/owncloud\/owncloud_data\/apps\/templateeditor\/controller\/adminsettingscontroller.php#31","level":3,"time":"2014-11-29T23:34:43+00:00","method":"GET","url":"\/index.php\/apps\/templateeditor\/settings\/mailtemplate?theme=default&template=core%2Ftemplates%2Fmail.php"}
{"app":"files","message":"Upload error: 3 - The uploaded file was only partially uploaded","level":3,"time":"2014-11-29T23:36:41+00:00"}
{"app":"files","message":"Upload error: 3 - The uploaded file was only partially uploaded","level":3,"time":"2014-11-30T13:54:09+00:00"}
```

#### Browser log
```
a) The javascript console log
Using //@ to indicate sourceMappingURL pragmas is deprecated. Use //# instead jquery-1.10.0.min.js:1
Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src https://owncloud.servername.com 'unsafe-eval'"). files
Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help http://xhr.spec.whatwg.org/ jquery-1.10.0.min.js:6
window.controllers is deprecated. Do not use it for UA detection. ace.js:1716
Use of getPreventDefault() is deprecated.  Use defaultPrevented instead. jquery-1.10.0.min.js:5

b) The network log 
Only getting an orange triangle on GET index.php, says its status Found 302

c) ...
```

**Comment by PVince81** [comment]
Any more luck with 8.0.6 or 8.1.1 ? It is likely that this issue is related to your environment/configuration.

Feel free to reopen if the issue still persist with the newer versions of OC.

**Comment by JSmith-BitFlipper** [comment]
I changed the router (for a different reason non-pertinent to owncloud) and the issue resolved itself. I used to have a rather old router before. It worked perfectly fine with everything, but for some reason it did not play nice with large owncloud uploads. I remember doing some (rather extensive) troubleshooting on my own and believe, now that with the new router all is fine, the issue is from the router getting overloaded or something.
